src/Snppts/Snippets/MauiMudBlazor.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace Snppts.Snippets;
+
+public class MauiMudBlazor : IAmASnippet
+{
+    public string Slug => "MauiBlazorWeb";
+    public string Title => "Maui Mud Blazor";
+
+    public GitHubRepoInfo GitHubRepoInfo => new GitHubRepoInfo("kevmoens/MauiBlazorWeb");
+
+    public bool ContainsAndroidSample => true;
+    public bool ContainsiOSSample => true;
+    public bool ContainsUWPSample => false;
+    public Sdk BuiltWith => Sdk.Maui;
+
+    public string Description => "";
```

Comment:
Could you maybe add a little description about what we are seeing here and why it is special? :)